Makefile

```diff
@@ -74,11 +76,19 @@ analyze: python
 
 .PHONY: format
 format: python
-	tox -q -e py36-format
+	@tox -qe py36-format
 
 PHONY: checkformatting
 checkformatting: python
-	tox -q -e py36-checkformatting
+	@tox -qe py36-checkformatting
+
+.PHONY: isort
+isort: python
+	@tox -qe py36-isort
+
+PHONY: checkisort
+checkisort: python
+	@tox -qe py36-isort --run-command "isort --recursive --quiet --check-only h tests"
 
```

Comment:
If import sorting and formatting were done in the same environment, you'd also have one less environment to build.